distiller/data/etls/extract_pdf.py

```diff
@@ -0,0 +1,54 @@
+import json
+import sys
+import smart_open
+import io
+from datetime import datetime
+
+from django.conf import settings
+
+from ..models.single_audit_db import Audit
+from ..models.pdf_extract import PDFExtract
+
+from ...gateways import files
+from ...extraction import nlp, pdf_utils
+from ...extraction.analyze import analyze
+from ...fac_scraper.models import FacDocument
+
+
+def setup():
+    return nlp.setup()
+
+
+def get_all_pdfs():
+    return FacDocument.objects.all().values_list('pk', flat=True)
+
+
+def process_audit_pdf(processor, pdf_id):
+    try:
+        document = FacDocument.objects.get(id=pdf_id)
+        pdf = files.input_file(f"{settings.LOAD_TABLE_ROOT}/pdfs/{document.file_name}", mode='rb')
```

Comment:
I think this should be `FAC_DOCUMENT_ROOT` instead but i haven't finished with scraping locally. Please correct me if I'm wrong.

Reply:
Actually I think it should be `FAC_DOCUMENT_DIR`

Reply:
Thanks! Fixed.